packages/solid-query/src/QueryClientProvider.tsx

```diff
@@ -80,7 +80,17 @@ export const QueryClientProvider = (
     },
     props,
   )
-  onMount(() => mergedProps.client.mount())
+  onMount(() => {
+    mergedProps.client.mount()
+
+    if (process.env.NODE_ENV !== 'production' && mergedProps.contextSharing) {
+      mergedProps.client
+        .getLogger()
+        .error(
+          `The contextSharing option has been deprecated and will be removed in the next major version`,
+        )
+    }
+  })
```

Comment:
@ardeora please let me know if this code is `solid` 😅

Reply:
Hahahaha sorry for the late reply but yes! This is looking great. Shouldn't be an issue. 
![lgtm](https://user-images.githubusercontent.com/45807386/199174733-3f872678-72f0-48c1-892d-676985ca5aee.gif)